includes/class-newspack-popups.php

```diff
@@ -941,23 +941,90 @@ public static function archive_campaign( $id, $status ) {
 		update_term_meta( $id, self::NEWSPACK_POPUPS_TAXONOMY_STATUS, $status ? 'archive' : '' );
 	}
 
+	/**
+	 * Remove the default category from the given post, if it's the only category applied to that post.
+	 *
+	 * @param int $post_id ID of the post.
+	 */
+	private static function remove_default_category( $post_id ) {
+		$default_category_id = (int) get_option( 'default_category', false );
```

Comment:
Casting to `int` with a `$default` set to `false` looks a bit confusing.

Reply:
Good call, updated the default to `0` in 692ac31.